### PR TITLE
feat(workflow): Move the create-mapsheet workflow into basemaps workflow.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: workflows/basemaps/cog-mapsheet.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+kind: WorkflowTemplate
+metadata:
+  name: create-cog-mapsheet
+  namespace: argo
+spec:
+  entrypoint: main
+  arguments:
+    parameters:
+      - name: layer
+        value: "104687"
```
Comment (paulfouquet) [inline workflows/basemaps/cog-mapsheet.yaml:12]: Is that a default value that you want to leave?

Comment (Wentao-Kuang) [reply workflows/basemaps/cog-mapsheet.yaml]: Yes, it is.

**Review by paulfouquet — APPROVED** [review]

File: workflows/basemaps/cog-mapsheet.yaml (hunk 1 above)
Comment (blacha) [reply workflows/basemaps/cog-mapsheet.yaml]: I wonder if there is a better way to handle these `layerIds` ? to be fair this specific `layerId`  wont ever change?

Comment (Wentao-Kuang) [reply workflows/basemaps/cog-mapsheet.yaml]: I think we can change this to anything when we submit the job. This is just the default layer id.

**Pull request merged**